### PR TITLE
LPS-199072 Date and Time field | Critical Accessibility Issue - Elements must only use supported ARIA attributes

### PR DESCRIPTION
**Pull request by larissacribeiro**
Related Issue: [LPS-199072](https://liferay.atlassian.net/browse/LPS-199072)

### CODE REVIEW

**Review by alinedoleron — CHANGES_REQUESTED** [review]
Hi @larissacribeiro, I made a comment. Thanks!

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/CheckboxMultiple/CheckboxMultiple.es.js [hunk 1]
```diff
@@ -135,9 +135,7 @@ const Main = ({
 }) => (
 	<FieldBase name={name} readOnly={readOnly} {...otherProps}>
 		<CheckboxMultiple
-			accessibleProps={{
-				'aria-required': otherProps.required,
-			}}
+			aria-required={otherProps.required}
```
[the remark this answers is not in the document]
Comment (larissacribeiro) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/CheckboxMultiple/CheckboxMultiple.es.js]: Hi @alinedoleron! no problem, thanks for your review, I've just updated it! 💟

**Comment by larissacribeiro** [comment]
ci:test:sf

**Review by alinedoleron — APPROVED** [review]
LGTM!

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:forms

**Comment by larissacribeiro** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:forms

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:forward

**Pull request closed without merging**